README.md

```diff
@@ -0,0 +1,49 @@
+# Leantime OmniSearch
+
+A plugin for Leantime that gives access to a plethora of shortcuts, to speed up 
+navigating the system.
+
+## Development
+
+This plugin requires an instance of Leantime running locally.
+
+Git clone this repo into app/Plugins as follows:
+
+```shell
+    cd app/Plugins
+    git clone https://github.com/ITK-Leantime/leantime-omnisearch /OmniSearch
```

Comment:
This will clone to a folder in the root of the file system (`/OmniSearch`).

Consider rewriting to

````suggestion
git clone https://github.com/ITK-Leantime/leantime-omnisearch app/Plugins/OmniSearch
````